include/LLVMSPIRVOpts.h

```diff
@@ -128,6 +135,7 @@ class TranslatorOpts {
   ExtensionsStatusMap ExtStatusMap;
   // SPIR-V to LLVM translation options
   bool GenKernelArgNameMD = false;
+  bool SPIRVMemToReg = false;
```

Comment:
Lines 137-142 describes SPIR-V to LLVM translation options

I suggest to move this new option somewhere out of this block and left a comment that this option affects LLVM IR regularization phase